src/radio-group/README.md

```diff
@@ -1,6 +1,29 @@
 # Visual Studio Code Radio Group
 
-The `vscode-radio-group` is a web component implementation of a [radio group](https://w3c.github.io/aria-practices/#radiobutton). While any DOM content is permissible as a child of the `vscode-radio-group`, only `vscode-radio` content and slotted content with a role of `radio` will receive keyboard support.
+The `vscode-radio-group` is a web component implementation of a [radio group](https://w3c.github.io/aria-practices/#radiobutton).
+
+![Radio group hero](/docs/assets/images/radio-group-hero.png)
+
+## Usage
+
+| ❌ Don't                                                                                                                 | ✅ Do                                                                                  |
+| ------------------------------------------------------------------------------------------------------------------------ | -------------------------------------------------------------------------------------- |
+| ![Radio buttons that indicate multiple options can be selected](/docs/assets/images/radio-dont-1.png)                    | ![Radio buttons with one clear possible selection](/docs/assets/images/radio-do-1.png) |
+| Don't use radio buttons when more than one selection is possible. Try using [checkboxes](../checkbox/README.md) instead. | Use radio groups for options that feature a single selection at a time.                |
+
+| ❌ Don't                                                                                    | ✅ Do                                                                                   |
+| ------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------- |
+| ![Radio buttons arranged in a horizontal orientation](/docs/assets/images/radio-dont-2.png) | ![Radio buttons arranged in a vertical orientation](/docs/assets/images/radio-do-2.png) |
+| Avoid using horizontal radio group layouts.                                                 | Arrange radio groups vertically to make it easy to scan and compare options.            |
+
+| ❌ Don't                                                                    | ✅ Do                                                                    |
+| --------------------------------------------------------------------------- | ------------------------------------------------------------------------ |
+| ![Radio options with multi-line text](/docs/assets/images/radio-dont-3.png) | ![Radio buttons with concise labels](/docs/assets/images/radio-do-3.png) |
+| Don't use long blocks of text.                                              | Use concise language to describe each option.                            |
```

Comment:
Compared to the checkbox example I think you could adjust this example to be even more concise (i.e. matching the checkbox example)

Checkbox:
<img width="470" alt="Screen Shot 2022-02-04 at 11 34 29 AM" src="https://user-images.githubusercontent.com/39639992/152592044-37b6f73f-36fc-47cb-a72b-955e398a77a3.png">

Radio group:
<img width="611" alt="Screen Shot 2022-02-04 at 11 35 30 AM" src="https://user-images.githubusercontent.com/39639992/152592102-2857c71d-aa07-42dd-a5cb-9986708260a8.png">